CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.46.10] - 2023-11-13
```

Comment:
since later you'll make container side change that requires this restli change, you'll need to bump the minor version here, so that users who adopt that container version but still use an older restli version will fail to build, forcing them to bump restli version too.

Reply:
Thanks for the suggestion. Have updated the `minor` version